MacOS X uses utmpx. Apple's patchset (patch-apple.diff) adds code
that calls /usr/bin/login to handle utmpx editing. Thus, we have no
need to use screen's utmp editing code. This patch disables
screen's utmp editing.

This also fixes the "/var/run/utmp: No such file or directory"
errors as a consequence of the above.

--- config.h.in	2014-04-26 12:58:46.000000000 +0200
+++ config.h.in	2014-05-01 21:36:54.000000000 +0200
@@ -192,14 +192,14 @@
  * If screen is installed with permissions to update /etc/utmp (such
  * as if it is installed set-uid root), define UTMPOK.
  */
-#define UTMPOK
+#undef UTMPOK
 
 /* Set LOGINDEFAULT to one (1)
  * if you want entries added to /etc/utmp by default, else set it to
  * zero (0).
  * LOGINDEFAULT will be one (1) whenever LOGOUTOK is undefined!
  */
-#define LOGINDEFAULT	1
+#undef LOGINDEFAULT
 
 /* Set LOGOUTOK to one (1)
  * if you want the user to be able to log her/his windows out.
@@ -215,7 +215,7 @@
  * Set CAREFULUTMP to one (1) if you want that users have at least one
  * window per screen session logged in.
  */
-#define LOGOUTOK 1
+#undef LOGOUTOK
 #undef CAREFULUTMP
 
 
